### PR TITLE
Proposed changes to feature/dash branch

### PR DESCRIPTION
**Pull request by bassosimone**
I cannot run `npm install` for ooni-wui, which fails like:

```
$ npm install
npm WARN prefer global node-gyp@3.6.0 should be installed with -g

> phantomjs-prebuilt@2.1.12 install /Users/sbasso/src/github.com/TheTorProject/ooni-wui/node_modules/phantomjs-prebuilt
> node install.js

module.js:487
    throw err;
    ^

Error: Cannot find module 'request-progress'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/sbasso/src/github.com/TheTorProject/ooni-wui/node_modules/phantomjs-prebuilt/install.js:9:23)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
npm WARN ooni-wui@2.2.3 license should be a valid SPDX license expression

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! phantomjs-prebuilt@2.1.12 install: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the phantomjs-prebuilt@2.1.12 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     $HOME/.npm/_logs/2017-07-19T11_15_54_411Z-debug.log
```

So, rather than committing on `feature/dash`, I've created a pull request.

Cc: @hellais

### CODE REVIEW

File: src/routes/Measurements/components/nettests/Dash.js [hunk 1]
```diff
@@ -94,7 +96,7 @@ export class DashDetails extends React.Component {
           <div className='col-xs-6'>
             <button className='btn btn-secondary' onClick={() => this.toggledAdvanced()}>
               <FormattedMessage
-                id='nettests.ndt.More'
+                id='nettests.dash.More'
```
Comment (hellais) [inline src/routes/Measurements/components/nettests/Dash.js]: You should not change this key as the string `More` is already translated for ndt and there is no need to have people translate `More` in the context of Dash again.

File: src/routes/Measurements/components/nettests/Dash.js [hunk 2]
```diff
@@ -132,7 +134,7 @@ export class DashDetails extends React.Component {
             <div className='col-xs-6'>
               <button className='btn btn-secondary' onClick={() => this.toggledAdvanced()}>
                 <FormattedMessage
-                  id='nettests.ndt.Less'
+                  id='nettests.dash.Less'
```
Comment (hellais) [inline src/routes/Measurements/components/nettests/Dash.js]: Same as above.

File: src/routes/Measurements/components/nettests/Dash.js [hunk 3]
```diff
@@ -47,13 +46,17 @@ const minimumBitrateForVideo = [
...
-    if (test_keys.simple.median_bitrate >= rate['sfr_min_bitrate']) {
+    // Make sure we select the lowest speed bucket in case they overlap
+    if (optimalRate === null &&
+        test_keys.simple.median_bitrate >= rate['sfr_min_bitrate']) {
       optimalRate = rate
     }
   })
+  if (optimalRate === null) {
+    optimalRate = minimumBitrateForVideo[0]
+  }
```
Comment (hellais) [inline src/routes/Measurements/components/nettests/Dash.js]: Isn't this equivalent to what I was doing?

File: src/util/nettest.js [hunk 4]
```diff
@@ -50,103 +50,6642 @@ export const getDeckIcon = (iconName, other) => {
...
+  "mzh": "TR", 
+  "mzi": "ML", 
+  "mzk": "KI", 
+  "mzl": "CO", 
+  "mzm": "FR", 
+  "mzo": "CU", 
+  "mzp": "NZ", 
+  "mzq": "ZA", 
+  "mzr": "AF", 
+  "mzs": "MY
```
Comment (hellais) [inline src/util/nettest.js]: If you are to add such a big dump of the airport codes maybe we can embed it in a smarter way, by building the map from the countrycode->airport map?

File: src/routes/Measurements/components/nettests/Dash.js (hunk 1 above)
Comment (bassosimone) [reply src/routes/Measurements/components/nettests/Dash.js]: Ah, yes, it makes sense. I thought it was not updated.

File: src/routes/Measurements/components/nettests/Dash.js (hunk 3 above)
Comment (bassosimone) [reply src/routes/Measurements/components/nettests/Dash.js]: See https://github.com/TheTorProject/ooni-wui/pull/53#discussion_r128340478

**Comment by bassosimone** [comment]
@hellais I have now updated the pull request. Then only bit left to discuss is the algorithm to be used to select the bucket, for which I commented in https://github.com/TheTorProject/ooni-wui/pull/53#discussion_r128340478.

Meanwhile, I guess I'll make a build for mobile and start testing it. It won't hurt.

**Comment by bassosimone** [comment]
For the records, I build the UI in https://github.com/TheTorProject/ooniprobe-android/pull/95/commits/093a2b6457ba5226ef48e0ccb2a98ee253e5a235 using 5beff8e.

File: src/routes/Measurements/components/nettests/Dash.js (hunk 3 above)
Comment (hellais) [reply src/routes/Measurements/components/nettests/Dash.js]: Replied

**Comment by hellais** [comment]
This looks good to me.

**Pull request merged**